src/encord_active/app/cli/main.py

```diff
@@ -86,11 +86,33 @@ def visualise(
     streamlit_page = (Path(__file__).parents[1] / "streamlit_entrypoint.py").expanduser().absolute()
     data_dir = project_path.expanduser().absolute().as_posix()
     sys.argv = ["streamlit", "run", streamlit_page.as_posix(), data_dir]
+    (Path.home() / ".streamlit").touch(exist_ok=True)  # Bypass Streamlit email question.
```

Comment:
[nitpick] 💅
There is no need to `touch` the file every time we run the `visualize` command. Should we perhaps check for existence instead of `exists_ok`?

```suggestion
    st_config = Path.home() / ".streamlit"
    if not st_config.exists()
        st_config.touch() # Bypass Streamlit email question.
```

Also, we might want to move this snippet to a method or decorator and call it on the `hello` as well.

Reply:
@Encord-davids should be resolved with latest commit.